schemas/lioncore.m3.serialization.schema.json

```diff
@@ -76,7 +76,7 @@
         },
         "concept": {
           "const": {
-            "metamodel": "LIonCore_M3",
+            "metamodel": "LionCore_M3",
```

Comment:
Outdated?

Reply:
If you don't mind, we go over the precise IDs and keys in a separate PR.